.github/workflows/docs-deploy.yml

```diff
@@ -83,3 +87,8 @@ jobs:
         if: ${{ success() && github.ref == 'refs/heads/master' }}
         run: |-
           gsutil -m rsync -d -R docs/build/html/ gs://lightning-docs-latest
+
+      # fixme
```

Comment:
What needs to be fixed here?

Reply:
:innocent: shall be removed, will do it in next PR, sorry